packages/shared_preferences/shared_preferences/README.md

```diff
@@ -45,3 +99,16 @@ You can populate `SharedPreferences` with initial values in your tests by runnin
 Map<String, Object> values = <String, Object>{'counter': 1};
 SharedPreferences.setMockInitialValues(values);
 ```
+
+### Storage location on platforms
+
+| Platform | Location |
+| :--- | :--- |
+| Android | SharedPreferences |
+| iOS | NSUserDefaults |
+| Linux | LocalFileSystem |
```

Comment:
I don't think the name of the class that abstracts the filesystem in Dart is a meaningful description. Saying "In the `XDG_DATA_HOME` directory" would be a lot more meaningful.